accepter.tf

```
@@ -55,25 +55,48 @@ data "aws_subnets" "accepter" {
     name   = "vpc-id"
     values = [local.accepter_vpc_id]
   }
+  dynamic "filter" {
+    for_each = var.accepter_subnet_tags
+    content {
+      name   = "tag:${filter.key}"
+      values = [filter.value]
+    }
+  }
```

```suggestion
  tags = var.accepter_subnet_tags
```
See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/subnets#tags

---

totally misread that documentation